### DISCUSSION TITLE
GraphQL error in Dagster UI during asset materialization or job run

### DISCUSSION THREAD

**Discussion by bruk0** [dagster-io/dagster#24287] [Q&A]
I have some bug in Dagster during job run related to GraphQL
The `install_requires`  from `setup.py`
install_requires=[
        "dagster",
        "dagster-cloud",
        "dagster-dbt",
        "dagster-embedded-elt",
        "dagster-gcp==0.24.0",
        "dagster-graphql==1.8.0",
        "dagster-pandas==0.24.0",
        "dagster-pipes==1.8.0",
        "dagster-webserver==1.8.0",
        "dlt==0.5.2",
        "google-api-core==2.19.1",
        "google-api-python-client==2.140.0",
        "google-auth==2.33.0",
        "google-auth-httplib2==0.2.0",
        "google-cloud-bigquery==3.25.0",
        "google-cloud-bigquery-storage==2.25.0",
        "google-cloud-core==2.4.1",
        "google-cloud-secret-manager==2.10.0",
        "google-cloud-storage==2.18.2",
        "dbt-core",
        "dbt-duckdb",
        "dbt-bigquery",
        "google-analytics-data",
        "google-api-python-client",
        "google-auth-oauthlib",
        "requests_oauthlib"
    ], 

The `requirements.txt`

setuptools~=68.2.0
dlt~=0.5.2
pendulum~=3.0.0
google-api-python-client~=2.140.0
dagster~=1.8.0
dagster
dagster-cloud
dagster-cloud-cli
dagster-dbt
dagster-embedded-elt
dagster-gcp==0.24.0
dagster-graphql==1.8.0
dagster-pandas==0.24.0
dagster-pipes==1.8.0
dagster-webserver==1.8.0
dlt==0.5.2
google-api-core==2.19.1
google-api-python-client==2.140.0
google-auth==2.33.0
google-auth-httplib2==0.2.0
google-cloud-bigquery==3.25.0
google-cloud-bigquery-storage==2.25.0
google-cloud-core==2.4.1
google-cloud-secret-manager==2.10.0
google-cloud-storage==2.18.2
dbt-core
dbt-duckdb
dbt-bigquery
google-analytics-data  
google-api-python-client  
google-auth-oauthlib  
requests_oauthlib
dlt[bigquery]>=0.3.25

Image
![image](https://github.com/user-attachments/assets/7abfacd9-8359-44ef-a2ae-3f68d4035ecd)

**Comment by bruk0** [comment]
Those error crashes the real time view of job run.

**Comment by garethbrickman** [comment]
What version of dagster are you using if you run `dagster -v`? 

Version 1.8.3 was yanked for known errors that resemble this one. You could try pin your dagster version to use `dagster==1.8.4`
